src/Transports/MassTransit.SqlTransport.SqlServer/SqlTransport/SqlServer/SqlServerDatabaseMigrator.cs

```diff
@@ -1239,6 +1239,7 @@ CREATE OR ALTER PROCEDURE {0}.RequeueMessages
     @queueName nvarchar(256),
     @sourceQueueType int,
     @targetQueueType int,
+    @messageCount int,
```

Comment:
I wonder if this is going to break if an existing 8.3 install is present since the signature has changed and they'd be forced to delete it.

Reply:
I guess I don't care at this point.

Reply:
Wo do `create or update` at the top.